test/e2e/test/context.go

```diff
@@ -64,7 +64,14 @@ func initializeContext() {
 	}
 
 	logutil.ChangeVerbosity(ctx.LogVerbosity)
+
+	// Do not print credentials in the following log output
+	backup := ctx.GCPCredentials
+	ctx.GCPCredentials = nil
+
 	log.Info("Test context initialized", "context", ctx)
+
+	ctx.GCPCredentials = backup
```

Comment:
This is very awkward and would go away if we re-used the existing `test-secrets` logic.